packages/jaeger-ui/src/vite-env.d.ts

```diff
@@ -14,3 +14,13 @@
 
 // eslint-disable-next-line spaced-comment
 /// <reference types="vite/client" />
+
+declare namespace global {
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  interface Window {
+    // dataLayer property is used by Google Tag Manager
+    // It should be only be changed with the help of the custom Interface WindowWithGATracking
+    // defined in utils/tracking/ga.ts
+    dataLayer: never;
+  }
+}
```

Comment:
Could you elaborate why this change is needed?

Reply:
> On second thoughts, the most ideal implementation would be for `dataLayer` to have the appropriate types within `ga.ts` and have the type `never` outside it so that no other module tries to access or modify the same. 

This had been my initial reasoning but I think it would be better to remove it considering the way you have advised to handle the types in the previous comment.